Mathlib/Algebra/Order/Monoid/Canonical/Defs.lean

```diff
@@ -175,6 +175,9 @@ end CanonicallyOrderedCommMonoid
 theorem pos_of_gt {M : Type*} [CanonicallyOrderedAddCommMonoid M] {n m : M} (h : n < m) : 0 < m :=
   lt_of_le_of_lt (zero_le _) h
 
+theorem pos_of_gt' {M : Type*} [Preorder M] [OrderBot M] {n m : M} (h : n < m) : ⊥ < m :=
```

Comment:
I think this was already discussed, but this theorem already exists as `bot_lt_of_lt`.